OsmAnd-java/src/main/java/net/osmand/router/RouteResultPreparation.java

```diff
@@ -1313,7 +1407,7 @@ private TurnType processRoundaboutTurn(List<RouteSegmentResult> result, int i, b
 			}
 		}
 		// combine all roundabouts
-		TurnType t = TurnType.getExitTurn(exit, 0, leftSide);
+		TurnType t = TurnType.getExitTurn(exit, 0, isClockwise(result));
```

Comment:
isClockwise(result) ? should be checking only rr.getObject() or roundabout - https://osmand.net/map?start=48.873450%2C2.296104&end=48.872960%2C2.294321&mode=pedestrian#18/48.87349/2.29507 test